blobstore/common/rpc/auditlog/decoder.go

```diff
@@ -60,7 +60,7 @@ var DefaultRequestHeaderKeys = []string{
 type DecodedReq struct {
 	Path   string
 	Header M
-	Params []byte
+	Params M
```

Comment:
`params` is request body which is not always map.!   we do not read the body, so define to string if fine.

Reply:
@setcy  need to fix this.